src/latis-python-client/latis.py

```diff
@@ -43,30 +43,34 @@ def __datasetWillUseVersion3(baseUrl, dataset, preferVersion2):
             return True
         except Exception:
             print("[WARN]: " + dataset +
-                  " cannot be accessed through Latis version 3." +
+                  " cannot be accessed through LaTiS version 3." +
                   " Auto switching to version 2.")
 
             return False
 
 
 def data(baseUrl, dataset, returnType,
          projections=[], selections=[], operations=[], preferVersion2=False):
-    """Retrieve dataset from Latis instance.
+    """
+    Retrieve dataset from LaTiS instance.
+
+    This method fetches the dataset data from a LaTiS instance and
+    transforms it into the specified format.
 
     Args:
-        baseUrl (str): The base URL of the Latis instance.
+        baseUrl (str): The base URL of the LaTiS instance.
         dataset (str): The name of the dataset.
         returnType (str): Desired return type ('NUMPY' or 'PANDAS').
-        projections (list, optional): List of projections to use.
-        selections (list, optional): List of selections to use.
-        operations (list, optional): List of operations to apply.
-        preferVersion2 (bool, optional): If True, prefer Latis version 2.
+        projections (list[list[str]], optional): List of projections to use.
```

Comment:
Aren't projections just lists of strings? Why'd you label them as lists of lists?

Reply:
Yep. Forgot `project()` appends each element not just the list.